brainstormforce/astra
# AST-2860 - Read Time option added to Meta options

### Description

- Added Read Time option to meta options.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2860

### Screenshots

https://d.pr/v/DgsgKz

### Types of changes


- bugfix - Read Time option missing from Single Post & CPT
- Modified dynamic single post structure's meta config to include Read Time option.
- Added backwards compatibility.



### How has this been tested?

- Customizer > Single Posts > Post title > Meta options.
- Read Time option should be visible & functional.
- Check for other post types (Pages, CPT's)
- Check for backwards compatibility.

### Checklist:
